### PR TITLE
Create full collection

### PR DESCRIPTION
**Pull request by evesonr**
- Running tool on a larger collection
- switching to cosine similarity
- concatenating all feedback responses per feedback session
- updating logo with new design
- other superficial changes

### CODE REVIEW

File: app/banner.html
Comment (hwrightson) [inline app/banner.html:1]: Comments need updating as they no longer match the values

Comment (hwrightson) [reply app/banner.html:1]: (lines 3 and 9)

File: prompts/openai_summarise.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 system_prompt = """
 You are a content and publishing expert working for a UK government department. Your task is to summarise a collection of user feedback to help identify common themes and issues. The feedback is submitted through the website www.gov.uk and is stored as free text. Your objective is to provide a concise summary of the top 3 themes of the feedback, highlighting the main topics and concerns raised by users. This summary will be used to inform the development and improvement of government digital services, ensuring they meet the needs of the public efficiently and effectively.
 Topics should relate to the content and services provided by the government, and should not include any personal or sensitive information. Your summary should be should be written in clear, plain English. Focus on topics that relate to actionable changes that a government department could make to improve the digital services they provide.
+Be specific about the topics in the feedback, and avoid generalisations. For example, instead of saying "tax" or "taxation", talk about the specific aspects of tax that they mention.
```
Comment (hwrightson) [inline prompts/openai_summarise.py:4]: Do we want to give an example? e.g. vehicle tax/car tax or income tax?

File: src/sql_queries.py [hunk 2]
```diff
@@ -41,10 +42,46 @@
...
+FROM
+    @EVALUATION_TABLE
+"""
+
+query_all_feedback = """
+WITH CTE AS (
+  SELECT feedback_record_id,
+  STRING_AGG(response_value, ', ') AS response_value_full
+FROM
+  `govuk-ai-publishing.joined_feedback_view.feedback_6mth_with_metadata_updated`
```
Comment (hwrightson) [inline src/sql_queries.py:58]: I don't think it's super important but should the table names be env variable names in this query string?

File: src/utils/call_openai_summarise.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def create_openai_summary(
         ]
         completion = client.chat.completions.create(
             messages=messages,  # type: ignore
-            max_tokens=250,
+            max_tokens=1000,
```
Comment (hwrightson) [inline src/utils/call_openai_summarise.py:21]: If we're upping this to 1k should be include something in the prompt that tries to ensure that it's as concise as possible? (Might already exist). Also are we happy with a temp of .2 when summarising 100 records?

**Review by hwrightson — APPROVED** [review]
Approved but with a couple of comments. The only important one to look at is probs regarding the sql queries and whether we're happy passing table names to git (I don't think it's a massive issue, but we use env vars elsewhere so might be best for security/consistency)

File: src/utils/call_openai_summarise.py (hunk 3 above)
Comment (evesonr) [reply src/utils/call_openai_summarise.py:21]: I think it couldn't hurt. I think we should be programmatic about it to ensure the summaries are useful - we could do some experimentation with prompts to ensure we get concise but meaningful summaries.

File: prompts/openai_summarise.py (hunk 1 above)
Comment (evesonr) [reply prompts/openai_summarise.py:4]: Let's do some experimentation with the system prompt once we have the product live and hosted?

File: src/sql_queries.py (hunk 2 above)
Comment (evesonr) [reply src/sql_queries.py]: This was an oversight. Fixed, good spot.

**Pull request merged**